tools/pipelines/build-test-tools.yml

```diff
@@ -85,7 +85,7 @@ extends:
     packageManagerInstallCommand: 'pnpm i --frozen-lockfile'
     packageManager: pnpm
     testCoverage: ${{ variables.testCoverage }}
-    tagName:
+    tagName: test-tools
```

Comment:
Would be nice to make this one private too, but definitely a separate task.

Reply:
It can't be private (easily) because there are dependencies on it across multiple release groups.

Reply:
Are there? I only see deps in the client release group. A few references to the string in build-tools which I think might not be too hard to handle. But yeah, no rush to make changes here.